src/pycell/impl_.rs

```diff
@@ -156,40 +160,180 @@ impl PyClassBorrowChecker for BorrowChecker {
 }
 
 pub trait GetBorrowChecker<T: PyClassImpl> {
-    fn borrow_checker(cell: &PyCell<T>) -> &<T::PyClassMutability as PyClassMutability>::Checker;
+    fn borrow_checker(
+        cell: &PyClassObject<T>,
+    ) -> &<T::PyClassMutability as PyClassMutability>::Checker;
 }
 
 impl<T: PyClassImpl<PyClassMutability = Self>> GetBorrowChecker<T> for MutableClass {
-    fn borrow_checker(cell: &PyCell<T>) -> &BorrowChecker {
+    fn borrow_checker(cell: &PyClassObject<T>) -> &BorrowChecker {
```

Comment:
```suggestion
    fn borrow_checker(class_object: &PyClassObject<T>) -> &BorrowChecker {
```
I think these should be now called `class_object` (or something like that), there are a few more of that